examples/movie_view_ratings/run_all_frameworks.py

```diff
@@ -33,7 +33,7 @@
 from absl import flags
 from apache_beam.runners.portability import fn_api_runner
 import pyspark
-from examples.movie_view_ratings.common_utils import *
+from common_utils import *
```

Comment:
`common_utils.py` is in the same directory as `examples/movie_view_ratings/run_all_frameworks.py`, so using `from common_utils import *` always works.
- However, if we do **not** include the root folder of the project to the `sys.path` (e.g. by setting `PYTHONPATH`), using `from examples.movie_view_ratings.common_utils import *` would **not** work.

I know PyCharm sets the `PYTHONPATH` automatically, but VSCode doesn't. So you'd need to create a `launch.json` or `.env`, or set the `PYTHONPATH` manually.
- I wanted the code to work in all cases, which is why I removed the specified path.

However, I see now that the `analysis` folder is also used in other scripts that are **not** in the same directory (e.g. `examples/restaurant_visits/run_without_frameworks_utility_analysis.py`)
- In this case, removing the specified path would **not** work.
- So we need to add the root folder to the `sys.path` and use the complete path in the `import`
- Since this is needed for the `analsys` folder, to keep consistency I'll go back to `from examples.movie_view_ratings.common_utils import *` in the commit above, as you suggested.

By the way, in our Dockerfile, it already adds the `/code` to the `sys.path`. So the `import` above works both with or without the specified path when using the containerized dev setup.

FYI, all the other `run_*.py` files in the `movie_view_ratings` folder are also using only `from common_utils import *`.
- Please let me know if you want me to change them to specify the path as well.
- Or maybe we should move `common_utils.py` to the `analysis` folder.